runtime/api/kotlinx-serialization-runtime.api

```diff
@@ -29,15 +30,15 @@ public abstract interface class kotlinx/serialization/CompositeDecoder {
 	public abstract fun decodeSerializableElement (Lkotlinx/serialization/SerialDescriptor;ILkotlinx/serialization/DeserializationStrategy;Ljava/lang/Object;)Ljava/lang/Object;
 	public abstract fun decodeShortElement (Lkotlinx/serialization/SerialDescriptor;I)S
 	public abstract fun decodeStringElement (Lkotlinx/serialization/SerialDescriptor;I)Ljava/lang/String;
-	public abstract fun decodeUnitElement (Lkotlinx/serialization/SerialDescriptor;I)V
 	public abstract fun endStructure (Lkotlinx/serialization/SerialDescriptor;)V
-	public abstract fun getContext ()Lkotlinx/serialization/modules/SerialModule;
+	public abstract fun getSerializersModule ()Lkotlinx/serialization/modules/SerialModule;
 	public abstract fun getUpdateMode ()Lkotlinx/serialization/UpdateMode;
 	public abstract fun updateNullableSerializableElement (Lkotlinx/serialization/SerialDescriptor;ILkotlinx/serialization/DeserializationStrategy;Ljava/lang/Object;)Ljava/lang/Object;
 	public abstract fun updateSerializableElement (Lkotlinx/serialization/SerialDescriptor;ILkotlinx/serialization/DeserializationStrategy;Ljava/lang/Object;)Ljava/lang/Object;
 }
 
 public final class kotlinx/serialization/CompositeDecoder$Companion {
+	public static final field DECODE_DONE I
```

Comment:
btw, why ABI duplicated in both interface and companion?

Reply:
Unfortunately, this is our compilation strategy for companions in  interfaces